src/Rx/Weno/NewRxData.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c )2019. Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Rx\Weno;
+
+require_once "../../interface/globals.php";
```

Comment:
drop above line

Reply:
dropped

Reply:
I don't see how $this->pid = $GLOBALS['pid']; is going to work if I drop the globals include statement.
I tried it without it and it did not work previously in 5.0.2x

Reply:
This is a class and will only be run from other scripts that already include globals.php (in this case in the transmitrx.php script).